scripts/cmake/CheckForDependencies.cmake

```diff
@@ -0,0 +1,72 @@
+macro(check_for_dependencies)
```

Comment:
Those files are just moved from scripts into scripts/cmake, right? Is there a way to see that in the pull request? I was just wondering when going through the list of deleted files, how to distinguish between moved and deleted files.

Reply:
In one commit they were just moved, yes. In the PR as a whole they were also changed, which is why they are not shown as move only.